experiments/train_test_overlap/train_test_proofpile.py

```diff
@@ -44,7 +43,6 @@
 )
 
 
-@ray.remote(runtime_env={"env_vars": {"JAX_PLATFORMS": "cpu", "PJRT_DEVICE": "cpu"}})
```

Comment:
worried this doesn't have the cpu stuff but maybe that's covered by the ResourceSpec stuff